pkg/minishift/update/update.go

```diff
@@ -78,6 +80,11 @@ func IsNewerVersion(localVersion, latestVersion semver.Version) bool {
 func Update(latestVersion semver.Version) error {
 	var extName string
 
+	path, _ := osext.Executable()
+	if writable := util.IsDirectoryWritable(path); !writable {
+		return fmt.Errorf("Directory '%s' containing binary doesn't have write permission.\nPlease fix the permission issue and try running the command again.", filepath.Dir(path))
+	}
```

Comment:
s/containing binary// i.e. `Directory '%s' doesn't have write permission.\nPlease fix the permission issue and try running the command again.`

Reply:
Sure

Reply:
Done